v8/src/base/platform/platform-starboard.cc

```diff
@@ -9,6 +9,7 @@
 #include <stdio.h>
 #include <unistd.h>
 
+#include "base/logging.h"
```

Comment:
v8 is actually an exception. V8 does not directly depend on Chromium base// ( v8 can be built completely standalone ). 

So this change should be undone.

Reply:
Done